pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -298,27 +297,26 @@ private static void start() throws MalformedURLException {
         Config sqlConfig = ConfigFactory.load(ConfigFile.SQL_CONFIG_FILE);
         initSqlCommands(sqlConfig);
 
-        if (cfg.hasPath(ConfigFile.DO_LIQUIBASE_IN_STUDY_SERVER) && cfg.getBoolean(ConfigFile.DO_LIQUIBASE_IN_STUDY_SERVER)) {
-            log.info("Running liquibase migrations in StudyServer against database");
-            LiquibaseUtil.runLiquibase(dbUrl, TransactionWrapper.DB.APIS);
-            LiquibaseUtil.releaseResources();
-        }
-        //@TODO figure out how to do this only at deployment time.
-        CacheService.getInstance().resetAllCaches();
-        TransactionWrapper.useTxn(TransactionWrapper.DB.APIS, LanguageStore::init);
-
+        // start responding to GAE _ah/start commands as soon as we can,
+        // even if that means we accept traffic before we've done
+        // cache cleanup and possibly lengthy liquibase updates
         if (appEnginePort != null) {
             port(Integer.parseInt(appEnginePort));
         } else {
             port(configFilePort);
         }
         threadPool(-1, -1, requestThreadTimeout);
         JettyConfig.setupJetty(preferredSourceIPHeader);
+        registerAppEngineCallbacks();
```

Comment:
Change here is order of operations: first, respond to GAE start/stop.  Then, boot stuff.